force-app/main/default/classes/DummyJSONCallout.cls

```diff
@@ -33,19 +33,47 @@ public with sharing class DummyJSONCallout {
 	 * 
      * @param dummyUserId The ID of the user in the external system to fetch data for.
      */
-
+	
 	public static void getDummyJSONUserFromId(String dummyUserId) {
 		// Create HTTP request to send.
+		Http http = new Http();
+		HttpRequest request = new HttpRequest();
 
 		// Set the endpoint URL. Use direct URL or for best practices use Named Credential.
+		request.setEndpoint('callout:DummyJsonUser.namedCredential' + dummyUserId);
 
 		// Set the HTTP method to GET.
+		request.setMethod('GET');
 
 		// Send the HTTP request and get the response.
-
+		HttpResponse response = http.send(request);
 		// If the HTTP response code is successful, parse the JSON response and update the contact.
-		Contact cont = null;
-		upsert cont DummyJSON_Id__c; //insert/update from the JSON response using the external id (dummyUserId)
+		if(response.getStatusCode() == 200){
+			String responseBody = response.getBody();
+
+			Map<String,Object> parsedResponse = (Map<String,Object>)JSON.deserializeUntyped(responseBody);
+			System.debug(parsedResponse);
+
+			Contact cont = new Contact();
+			cont.FirstName = (String)parsedResponse.get('firstName');
+			cont.LastName = (String)parsedResponse.get('lastName');
+			Map<String,Object> addressMap = (Map<String,Object>)(parsedResponse.get('address'));
+			cont.MailingStreet = (String)addressMap.get('address');
+			cont.MailingCity = (String)addressMap.get('city');
+			cont.MailingState = (String)addressMap.get('state');
+			cont.MailingCountry = (String)addressMap.get('country');
+			cont.MailingPostalCode = (String)addressMap.get('postalCode');
+			cont.Birthdate = Date.valueOf((String)parsedResponse.get('birthDate'));
+			cont.Email = (String)parsedResponse.get('email');
+			cont.DummyJSON_Id__c = String.valueOf(parsedResponse.get('id'));
+			cont.Phone = (String)parsedResponse.get('phone');
```

Comment:
Do some error checking or defaulting the values if some of the values come back as null.